blueocean-dashboard/reports/junit.xml

```diff
@@ -8,11 +8,11 @@
     </testcase>
```

Comment:
why is this file in the repo? (i see it wasn't _added_)

Reply:
Ew.